package.json

```diff
@@ -84,6 +84,6 @@
     "prerender": "SCRIVITO_PRERENDER=true npm run build && node buildNode/prerender_content.js",
     "start": "webpack serve",
     "eslint": "eslint --max-warnings 0 *.js src/",
-    "es-check": "npm run build && es-check es5 --not=build/assets/scrivito_extensions. './build/**/*.js'"
+    "es-check": "npm run build && es-check es5 --not='build/assets/chunk-*' './build/**/*.js'"
```

Comment:
* [ ] This is dangerous. We already use `@loadable/component` for widgets (`ThumbnailGalleryComponent`), and it's highly recommendable for more extensive use. However, these widgets must still be ES-checked because the code is relevant for *visitors*.